packages/auth/src/commands/auth/2fa/disable.ts

```diff
@@ -15,8 +15,13 @@ Disabling 2fa on me@example.com... done`
 
   async run() {
     ux.action.start('Disabling 2fa')
-    const {body: account} = await this.heroku.get<Heroku.Account>('/account')
+    const {body: account} = await this.heroku.get<Heroku.Account>('/account'), {
+        headers: {
+          Accept: 'application/vnd.heroku+json; version=3.with_vaas_info',
```

Comment:
this is the variant Dashboard uses to get vaas enrollment status: https://github.com/heroku/dashboard/commit/c7ef8f7361132a8db14cbd8757e99eaf2b752a12